telegraf/README.md

```diff
@@ -0,0 +1,223 @@
+
+_This is a directory that consolidates all the metadata associated with the SignalFx Telegraf Agent. The code repository for this project can be found [here](https://github.com/signalfx/telegraf/)._
+
+- [Description](#description)
+- [Requirements and Dependencies](#requirements-and-dependencies)
+- [Features](#features)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [License](#license)
+
+### DESCRIPTION
+
+[Telegraf](https://github.com/influxdata/telegraf) is an open source daemon that collects statistics from a system and publishes them to a destination of your choice. You can use Telegraf to monitor infrastructure metrics, and enable Telegraf plugins that monitor a wide range of software. 
+
+#### FEATURES
+
+Sending data using Telegraf allows you to take advantage of the following features:
+
+- The [SignalFx Infrastructure page](http://docs.signalfx.com/en/latest/built-in-content/host-nav.html) visualizes hosts that are monitored using the SignalFx Telegraf Agent.
+
+  [<img src='./img/telegrafhostspage.png' width=200px>](./img/telegrafhostspage.png)
+
+  [<img src='./img/hostspagesinglehost.png' width=200px>](./img/hostspagesinglehost.png)
+
+- SignalFx provides [recommended detectors](http://docs.signalfx.com/en/latest/built-in-content/recommended-detectors.html) for hosts instrumented with the SignalFx Telegraf Agent. These built-in templates allow you to instantly create intelligent detectors based on SignalFx’s powerful analytics.
+
+- The SignalFx metadata plugin for Telegraf is a plugin that enriches your data by sending metadata about your hosts to SignalFx. This plugin is included by default in SignalFx’s Telegraf Agent.
+
+### REQUIREMENTS AND DEPENDENCIES
+
+The SignalFx Telegraf Agent is supported on the following operating systems:
+
+| Operating System  | Version        |
+|-----------|----------------|
+| Amazon Linux | 2014.09, 2015.03, 2015.09, & 2016.03 |
+| Debian  | 7 & 8 |
+| RHEL/Centos | 6.x & 7.x |
+| Ubuntu  | 12.04, 14.04, 15.04 & 16.04 |
+
+### INSTALLATION
+
+#### Install on a host
+
+*  Download the latest release for your platform from the [SignalFx Telegraf github repo](https://github.com/signalfx/telegraf/releases).
+*  Unzip the zip file.
+```bash
+unzip Linux-x86_64.zip
+```
+*  Copy the binary `telegraf` to the location of your choosing.
+
+### CONFIGURATION
+
+The SignalFx Telegraf Agent must be configured before it is run.  The agent binary is capable of generating a configuration file.
+
+* To generate a configuration file, execute the `telegraf` binary with the option `config` and redirect it to a file named `telegraf.conf`
+```bash
+./telegraf config > telegraf.conf
+```
+*  In `telegraf.conf` optionally configure the `[agent]` block.  Specify where the Telegraf `logfile` should be written, and optionally specify the `hostname` to the desired setting.
+```
```

Comment:
In this instruction, we direct the user to change two things -- and then include 60 lines of extraneous config, without even highlighting what we want the user to do. This is true for the following steps as well. We need to tighten this up to give simple and clear instructions. In other integrations we provide a table of config parameters rather than literal configuration file. If this entire config section literally needs to be copy/pasted in, let's talk about a better way of providing it.